src/formatDate.js

```diff
@@ -50,7 +50,37 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(`${fromFormat[3]}`);
```

Comment:
I suggest that it anyway is a string 

```suggestion
  const dateArr = date.split(fromFormat[3]);
```